### ISSUE TITLE
Good job for your work

### ISSUE THREAD

**Issue by zhouyaguang** [mertkosan/DyGED#1]
The algorithm proposed has made significant contributions to the research of event detection and anomaly detection, but the codes are insufficient to reproduce. I am looking forward to you providing complete codes for learning and research. Thank you very much for your work!

**Comment by mertkosan** [comment]
Thank you so much for your comments! We will try to share the cleaned training procedure as soon as possible, by the time being, please refer to the following information:

1) Main class is DyGED which is an example of creation in models.py
2) The forward function takes node features (X) and adjacency matrix (A)
2.1) X includes (batch_size, number_of_nodes, number_features)
2.2) A includes (batch_size, number_of_nodes, number_of_nodes)
3) We provide data and data_utils.py to create PyTorch geometric Data creation, however since our model is outdated and does not work with the current PyTorch Geometric pipeline yet, it needs to be changed to full feature matrix and adjacency matrices. (We may hopefully renovate our code to PyTorch Geometric functionalities later too)
4) The training pipeline is the same as other PyTorch implementations (defining loss function, optimizer, etc.)

Let us know if you have further questions.
